sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/sender.py

```diff
@@ -188,8 +188,8 @@ def send_transfer(self, message, *, send_async=False, **kwargs):
     def cancel_transfer(self, delivery):
         try:
             index = self._pending_deliveries.index(delivery)
-        except ValueError:
-            raise ValueError("Found no matching pending transfer.")
+        except ValueError as exc:
```

Comment:
keep or remove -- don't know if it adds anything

Reply:
I would probably raise `from None` here - but not overly fussed either way.

Reply:
It is from None now :)